### ISSUE TITLE
Labels should be associated with the form fields

### ISSUE THREAD

**Issue by SaptakS** [freedomofpress/taps.pressfreedomtracker.us#29]
The labels and the form fields in the filter component are not associated. Hence, the input fields don't actually have an associated label that is used by assistive technologies.
